exercises/isogram/src/example/java/Isogram.java

```diff
@@ -0,0 +1,37 @@
+package exemple;
+
+import java.util.HashSet;
+import java.util.Set;
+import static java.util.Arrays.stream;
+import static java.util.stream.Collectors.joining;
+
+public class Isogram {
```

Comment:
I am thinking that the semantics are very close on this class, but not quite right.  Think of it this way: what is the purpose of this class?  What do the methods do?  What if, for example:
```Java
IsogramChecker isogramChecker = new IsogramChecker();
String word = "isogram";
boolean result = isogramChecker.isIsogram(word);
```